README.md

```diff
@@ -197,6 +228,8 @@ After logging out, Auth0 redirects back to your app. Again, it needs to know you
 
 ## Persisting login after refresh
 
+**NB Make sure you're not blocking blocking cookies! Extensions like privacy badger will prevent Auth0 from setting cookies so refreshing between logins wont work**
```

Comment:
@Swizec realised I wrote blocking twice here!

Reply:
I'll fix it :)